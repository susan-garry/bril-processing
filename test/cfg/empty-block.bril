@main(n: int) {
	zero: int = const 0;
	cond: bool = eq n zero;
	br cond .end .rec;
.rec:
	v1: int = div n two;
	print v1;
.end:
}