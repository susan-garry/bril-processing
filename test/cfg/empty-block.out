{'block0': ['end', 'rec'], 'rec': ['end'], 'end': []}
func:  main
  block0: ['end', 'rec']
  rec: ['end']
  end: []
